enterprise/server/remote_execution/platform/platform.go

```diff
@@ -696,6 +700,18 @@ func FindValue(platform *repb.Platform, name string) string {
 	return value
 }
 
+// FindValueInTask scans the platform properties of the given task for the given
+// property name (ignoring case) and returns the value of that property if it
+// exists, otherwise "".
+func FindValueInTask(task *repb.ExecutionTask, name string) string {
+	p := task.GetAction().GetPlatform()
+	if p == nil || len(p.GetProperties()) == 0 {
+		p = task.GetCommand().GetPlatform()
+	}
+	value, _ := findValue(p, name)
+	return value
+}
```

Comment:
nit: I think if we had a helper func like `platform.GetProto(action, command)` that returns the effective platform, then we could instead write `p := platform.GetProto(task.GetAction(), task.GetCommand())` and `platform.FindValue(p, "some-prop")` instead of using this helper func. It's slightly more verbose but helps minimize the API surface of this package - and we only need this logic in a couple of places.